data/net/minecraft/client/model/geom/builders/CubeDefinition.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/client/model/geom/builders/CubeDefinition
+	METHOD <init> (Ljava/lang/String;FFFFFFFFLnet/minecraft/client/model/geom/builders/CubeDeformation;ZFF)V
+		ARG 1 comment
+		ARG 2 texCoordU
+		ARG 3 texCoordV
+		ARG 4 originX
+		ARG 5 originY
+		ARG 6 originZ
+		ARG 7 dimensionX
+		ARG 8 dimensionY
+		ARG 9 dimensionZ
+		ARG 10 cubeDeformation
```

Comment:
According to convention, this should follow the name of the field which the parameter is directly assigned to: `grow`.